en/02_Development_environment.adoc

```diff
@@ -2,454 +2,416 @@
-
-[,make]
-----
-CFLAGS = -std=c++17 -O2
-----
-
-We're going to use modern C{pp} (`pass:c[-std=c++17]`), and we'll set optimization level to O2.
-We can remove -O2 to compile programs faster, but we should remember to place it back for release builds.
-
-Similarly, define the linker flags in a `LDFLAGS` variable:
-
-[,make]
-----
-LDFLAGS = -lglfw -lvulkan -ldl -lpthread -lX11 -lXxf86vm -lXrandr -lXi
-----
-
-The flag `-lglfw` is for GLFW, `-lvulkan` links with the Vulkan function loader and the remaining flags are low-level system libraries that GLFW needs.
-The remaining flags are dependencies of GLFW itself: the threading and window management.
-
-It is possible that the `Xxf86vm` and `Xi` libraries are not yet installed on your system.
-You can find them in the following packages:
+=== Setting up CLion (optional)
 
-[,bash]
+You can get http://jetbrains.com[CLion] from there.  We recommend installing
+from the jetbrains toolbox so it can keep CLion up to date automatically.  To
+ use an IDE like CLion, we have to setup the environment variables that are
+ otherwise setup by when the terminal executes
+[,shell]
 ----
-sudo apt install libxxf86vm-dev libxi-dev
+source ~/vulkanSDK/default/setup-env.sh
 ----
-
-or
-
+To do that, open Settings, then select "Build, Execution, Deployment" and
+then select CMake. At the bottom of that window will be the environment
+variable, Just, add VULKAN_SDK=<fullPathToVulkanSDK> there and Vulkan will be
+found during compile time.  As a convenience, for runtime at least, we
+recommend placing the layers system wide.  To do that, from the terminal do
+this:
 [,bash]
 ----
-sudo dnf install libXi-devel libXxf86vm-devel
+sudo cp $VULKAN_SDK/lib/libVkLayer_*.so /usr/local/lib/
+sudo mkdir -p /usr/local/share/vulkan/explicit_layer.d
+sudo cp $VULKAN_SDK/share/vulkan/explicit_layer.d/VkLayer_*.json /usr/local/share/vulkan/explicit_layer.d
 ----
 
-or
+Alternatively, you could add VK_LAYER_PATH to your system environment
+variables, and point it to `$VULKAN_SDK/share/vulkan/explicit_layer.d` Also,
+you'd want to add to LD_LIBRARY_CONFIG the `$VULKAN_SDK/lib` path.  This is
+all done for you by the setup-env.sh file when using the terminal.
 
-[,bash]
-----
-sudo pacman -S libxi libxxf86vm
-----
+=== Setting up a CMake project
 
-Specifying the rule to compile `VulkanTest` is straightforward now.
-Make sure to use tabs for indentation instead of spaces.
+Now that you have installed all the dependencies, we can set up a basic
+CMake project for Vulkan and write a little bit of code to make sure that
+everything works.
 
-[,make]
-----
-VulkanTest: main.cpp
-	g++ $(CFLAGS) -o VulkanTest main.cpp $(LDFLAGS)
-----
+I will assume that you already have some basic experience with CMake, like
+how variables and rules work. If not, you can get up to speed very quickly with [this tutorial](https://cmake.org/cmake/help/book/mastering-cmake/cmake/Help/guide/tutorial/).
```

Comment:
Same as above. This link is using the markdown format, and needs to be changed to asciidoc link format.